wrangler-unenv-polyfills/wrangler.toml

```diff
@@ -1,4 +1,4 @@
 name = "unenv-worker"
 main = "worker.mjs"
-compatibility_flags = ["experimental:nodejs_compat_v2"]
-compatibility_date = "2024-08-29"
\ No newline at end of file
+compatibility_flags = ["nodejs_compat_v2", "nodejs_zlib"]
```

Comment:
hmmm.... this is not right, @anonrig said that zlib is now unflagged.

Yagiz, going on here? Can you please take a look and confirm that this is expected?

Reply:
`nodejs_zlib` is not needed. After 09/23, it is implied with nodejs_compat_v2

Reply:
It might be linked to the issue chased by Pete